modules/functions/src/models/FunctionExecutions.schema.ts

```diff
@@ -8,8 +8,9 @@ import { ConduitActiveSchema } from '@conduitplatform/module-tools';
 
 const schema: ConduitModel = {
   _id: TYPE.ObjectId,
-  functionName: {
-    type: TYPE.String,
+  function: {
```

Comment:
change the name of the field, function is a reserved name in JS and can create issues